scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -124,17 +111,125 @@ task CreateAndTieOutVds {
 
         python3 ./hail_gvs_import.py \
             --avro-path ${AVRO_PATH} \
-            --vds-path ${VDS_PATH} \
+            --vds-path ~{vds_destination_path} \
             --temp-path ${TEMP_PATH} \
             --references-path ${REFERENCES_PATH} \
             ~{true='' false='--use-vqsr-lite' use_classic_VQSR}
 
-        export JOINED_MATRIX_TABLE_PATH=${WORK}/joined.mt
+    >>>
+    runtime {
+        # `slim` here to be able to use Java
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-slim" ## should this be our regular docker now?
```

Comment:
I believe Hail requires Java to create the VDS and the comment on line 121 implies we need slim to be able to use Java, though I'm happy to be proven wrong on this if you know otherwise. 🙂 
```suggestion
        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-slim"
```

Reply:
hmmmmm I suppose....